src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs

```diff
@@ -66,6 +64,14 @@ public bool Equals(T[] x, T[] y)
                 }
                 for (int i = 0; i < x.Length; i++)
                 {
+                    if (x[i] == null)
```

Comment:
Yeah, now that the code got complex enough (after several review comments), I probably should :-). I will do it opportunistically with a next commit.